packages/docusaurus-theme-classic/src/theme/DocBreadcrumbs/index.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {
+  ThemeClassNames,
+  useDocsBreadcrumbs,
+  useThemeConfig,
+} from '@docusaurus/theme-common';
+import styles from './styles.module.css';
+import clsx from 'clsx';
+import {useLocation} from '@docusaurus/router';
+
+export default function DocBreadcrumbs(): JSX.Element | null {
+  const {pathname} = useLocation();
+  const breadcrumbs = useDocsBreadcrumbs();
+  const {breadcrumbs: enabled} = useThemeConfig();
+
+  function isExact(
+    items: {
+      href?: string;
+    }[],
+  ) {
+    return (
+      items.length === 1 &&
+      items[0].href?.replace(/\/$/, '') === pathname.replace(/\/$/, '')
+    );
+  }
+
+  if (
```

Comment:
We'd like to avoid putting such logic in the theme. 

I'd rather encapsulate it directly inside theme-common, considering it's part of the docs breadcrumb feature.

The current hook (not using themeConfig and applying this logic) could be renamed as `useSidebarBreadcrumbs`